README.md

```diff
@@ -65,23 +65,39 @@ make vendor
 
 **Config**
 
-A broker is configured via a `$ENV.config.yaml` file. Example files can be
+A broker is configured via a `config.yaml` file. Example files can be
 found under `etc/`. It's recommended to simply copy over `etc/ex.dev.config.yaml`
-to `etc/dev.config.yaml`, and edit as desired. `scripts/runbroker.sh` should
-handle providing the location to this file. Of course, this can be customized
-or the configuration file can be specified by cli args as well.
+to `etc/config.yaml`, and edit as desired. 
+
 
-## Targets
 
+## Targets
+### Broker Targets
```

Comment:
This might be too much, but do you think we could sort some of these into a developer targets section?